web-registry/src/pages/CreditClassDetails/CreditClassDetailsSimple/hooks/useCreditClassDisplayName.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+import { CreditClassMetadataLD } from 'generated/json-ld';
+
+export function useCreditClassDisplayName(
```

Comment:
maybe a utility function instead of a hook?

Reply:
I liked the hook here because it only has to run when something updates, instead of each render.